tests/unittests/distros/test_manage_service.py

```diff
@@ -3,6 +3,8 @@
 from tests.unittests.helpers import CiTestCase, mock
 from tests.unittests.util import MockDistro
 
+from . import _get_distro
```

Comment:
Nit: I think we prefer absolute imports these days.

Reply:
most other files in that directory import it that way ;)

but I'll happily fix that (and the others if desired)

Reply:
see #1760